integration-tests.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023-present Datadog, Inc.
+
+#set -x
+
+#export DD_TRACE_DEBUG=true
+fail=0
+failures=""
+
+testname="$1"
+
+fail() {
+    echo FAIL: $1
+    fail=1
+    failures+="FAIL: ${1}\n"
+}
+
+pass() {
+    echo PASS: $1
+    failures+="PASS: ${1}\n"
+}
+
+## Build Orchestrion
+go build -o ./_integration-tests/orchestrion
+
+cd ./_integration-tests
+
+## We need to change the default behavior of getting the latest release
+## and instead use the head of the main branch.
+go get github.com/datadog/orchestrion@main
```

Comment:
Shouldn't we import the local orchestrion to make sure we test the new changes on the branch? We can use `replace` in the go.mod, something like `echo "replace github.com/datadog/orchestrion => <path to the local orchestrion directory> " >> go.mod`

Reply:
See how we do it in parametric-tests in dd-trace-go https://github.com/DataDog/dd-trace-go/blob/8868d37cff43e00caaed80733d55a54c3008a9f0/.github/workflows/parametric-tests.yml#L43